### PR TITLE
style: replace back arrow with logo

### LINKED ISSUE #7100: Standups: change back arrow icon for Parabol's logo
Let's consider changing the back arrow icon located on the top left of the standup view, to the Parabol logo mark. It will not only look nicer, it can also help us with brand awareness.

From this:

image

To this:

image

Hovering the logo should display a tooltip that says: "Go back to your Dashboard".

### PR DESCRIPTION
**Pull request by enriquesanchez**
# Description

Fixes #7100

## Demo

**Before:**
Captura de Pantalla 2022-08-31 a la(s) 5 35 12 p m

**After:**
Captura de Pantalla 2022-08-31 a la(s) 5 35 25 p m

## Final checklist

- [ ] I checked the [code review guidelines](../docs/codeReview.md)
- [ ] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [ ] I have performed a self-review of my code, the same way I'd do it for any other team member
- [ ] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [ ] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [ ] PR title is human readable and could be used in changelog

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
+export PATH="/Users/enriquesanchez/.nvm/versions/node/v16.14.0/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin"
```
Comment (BartoszJarocki) [inline .husky/pre-commit:4]: -2 there has to be a different solution for the problems you're having with `yarn`. adding this user-specific `PATH` will cause this on every other machine
```
❯ git commit -m "test"
.husky/pre-commit: line 6: yarn: command not found
```

**Review by BartoszJarocki — CHANGES_REQUESTED** [review]
the code looks good, but we can't move forward with changes to `.husky/pre-commit` as it'll break the env for everyone else, let's figure it out!

File: .husky/pre-commit (hunk 1 above)
Comment (enriquesanchez) [reply .husky/pre-commit:4]: Oh! I did not mean to include that change here!!! 🙈 

@BartoszJarocki Is there a way I can remove it from this PR?

Comment (enriquesanchez) [reply .husky/pre-commit]: @BartoszJarocki Ok, I saw that I could revert and commit again. Let me know how all looks to you now 👍

**Review by BartoszJarocki — APPROVED** [review]

**Pull request merged**